packages/webapp/src/components/FilterDropdown/Markup.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { MenuItem, CssBaseline } from '@material-ui/core';
+
+import { FormContainer, SelectPreview, CircularProgressStyled } from './styled';
+
+const callSpinner = () => <CircularProgressStyled size={20} thickness={2.5} />;
+
+const callMenuItems = (options, loading) => {
+  if (loading) {
+    return (
+      <MenuItem value="boom" displayEmpty name="something">
+        {callSpinner()}
+      </MenuItem>
+    );
+  }
+
+  return options.map(({ id: idVal, name }) => {
+    const selectedKey = options.findIndex(({ id }) => id === idVal);
```

Comment:
What does selectedKey do? Why not use the `id` as the key?

Reply:
removed